backend/migrations/2023-07-15-152628_squashed_setup/up.sql

```diff
@@ -0,0 +1,461 @@
+    'enough',
+    'more than enough'
+);
+
+CREATE TYPE relation_type AS ENUM (
+    'companion',
+    'neutral',
+    'antagonist'
+);
+
+CREATE TYPE shade AS ENUM (
+    'no shade',
+    'light shade',
+    'partial shade',
+    'permanent shade',
+    'permanent deep shade'
+);
+
+CREATE TYPE soil_ph AS ENUM (
+    'very acid',
+    'acid',
+    'neutral',
+    'alkaline',
+    'very alkaline'
+);
+
+CREATE TYPE soil_texture AS ENUM (
+    'sandy',
+    'loamy',
+    'clay',
+    'heavy clay'
+);
+
+CREATE TYPE soil_water_retention AS ENUM (
+    'well drained',
+    'moist',
+    'wet'
+);
+
+CREATE TYPE water_requirement AS ENUM (
+    'dry',
+    'moist',
+    'wet',
+    'water'
+);
+
+CREATE FUNCTION check_layer_type() RETURNS trigger
+    LANGUAGE plpgsql
+    AS $$
+BEGIN
+    IF (SELECT type FROM layers WHERE id = NEW.layer_id) != 'plants' THEN
+        RAISE EXCEPTION 'Layer type must be "plants"';
+    END IF;
+    RETURN NEW;
+END;
+$$;
+
+
+CREATE TABLE layers (
+    id integer NOT NULL,
+    map_id integer NOT NULL,
+    type layer_type NOT NULL,
+    name text NOT NULL,
+    is_alternative boolean NOT NULL
+);
+
+
+CREATE SEQUENCE layers_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+CREATE TABLE maps (
+    id integer NOT NULL,
+    name text NOT NULL,
+    creation_date date NOT NULL,
+    deletion_date date,
+    last_visit date,
+    is_inactive boolean NOT NULL,
+    zoom_factor smallint NOT NULL,
+    honors smallint NOT NULL,
+    visits smallint NOT NULL,
+    harvested smallint NOT NULL,
+    privacy privacy_options NOT NULL,
+    description text,
+    location geography(Point,4326),
+    owner_id uuid NOT NULL
+);
+
+CREATE SEQUENCE maps_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+CREATE TABLE plants (
+    id integer NOT NULL,
+    unique_name text NOT NULL,
+    common_name_en text[],
+    common_name_de text[],
+    family text,
+    genus text,
+    edible_uses_en text,
+    --medicinal_uses text,
+    material_uses_and_functions text,
+    botanic text,
+    --material_uses text,
+    functions text,
+    heat_zone smallint,
+    shade shade,
+    soil_ph soil_ph[],
+    soil_texture soil_texture[],
+    soil_water_retention soil_water_retention[],
+    --environmental_tolerances text[],
+    --native_geographical_range text,
+    --native_environment text,
+    ecosystem_niche text,
+    deciduous_or_evergreen deciduous_or_evergreen,
+    herbaceous_or_woody herbaceous_or_woody,
+    life_cycle life_cycle[],
+    growth_rate growth_rate[],
+    height plant_height,
+    fertility fertility[],
+    --flower_colour text,
+    flower_type flower_type,
+    created_at timestamp without time zone DEFAULT now() NOT NULL,
+    updated_at timestamp without time zone DEFAULT now() NOT NULL,
+    has_drought_tolerance boolean,
+    tolerates_wind boolean,
+    plant_references text[],
```

Comment:
update_at can be disabled, but it is filled with data 100% because the insert_plants scraper updates these values when it changes values, i would keep it.
in `src/model/entity.rs`:
has_drought_tolerance is declared as used in hydrology layer
tolerates_wind is declared as used in wind layer

Reply:
i will comment out tolerates_wind for now.

Reply:
ok, lets keep it. `tolarates_wind` is needed for wind layer.